### PR TITLE
ICD10GM script refresh

### PR DESCRIPTION
**Pull request by divakhnenko**
Manual work was updated.
Insertion for valid_start_date, valid_end_date into concept_relationship_stage was changed in load_stage
New steps were added into readme

### CODE REVIEW

File: ICD10GM/load_stage.sql [hunk 1]
```diff
@@ -134,6 +137,11 @@ FROM concept_stage c1,
 WHERE c2.concept_code LIKE c1.concept_code || '%'
 	AND c1.concept_code <> c2.concept_code;
 
+/* Due to the similarity of concept_codes in ICD10 and ICD10GM inaccurate mapping can occur in ICD10GM. 
+For these cases necessary mapping was put into concept_relationship_manual table as well as deprecated old one. 
+Thus, these pares should be deleted from concept_relationship_stage on this stage */
+DELETE FROM concept_relationship_stage WHERE concept_code_1 IN (SELECT concept_code_1 FROM concept_relationship_manual) AND vocabulary_id_2 = 'SNOMED';
```
Comment (dimshitc) [inline ICD10GM/load_stage.sql:143]: do we still need this?
as you already added those relationships as 'D' in concept_relationship_manual

File: ICD10GM/manual_work/pre_load_stage.sql [hunk 2]
```diff
@@ -1,3 +1,7 @@
+-- delete from concept_manual table "dead" concepts
```
Comment (dimshitc) [inline ICD10GM/manual_work/pre_load_stage.sql:1]: only this query is actualy run, other queries you shouldn't run as concepts are already there

**Review by dimshitc — CHANGES_REQUESTED** [review]
In general:
go trough your readme and try to execute what you wrote

File: ICD10GM/load_stage.sql (hunk 1 above)
Comment (divakhnenko) [reply ICD10GM/load_stage.sql:143]: Yes, we still need this. Otherwise, there will be alive duplicates for the dead relationship.

Comment (divakhnenko) [reply ICD10GM/load_stage.sql]: You are right. This step was useless due to update in manual stages. 
They were put in GoogleDrive and link to them can be found in readme

**Review by dimshitc — CHANGES_REQUESTED** [review]
Again, you need to add your algorithm of manual work
and also add steps to readme:
- add English translations to concept_manual
- edit concept_relationship_manual according to new concepts in ICD10GM, or changesin SNOMED
and then add these in the folder 'manual_work'

**Review by dimshitc — APPROVED** [review]

**Review by hardhouse — APPROVED** [review]

**Pull request merged**